components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -97,8 +101,10 @@ open class GleanInternalAPI internal constructor () {
         // API. For this reason we're safe to set `initialized = true` right after it.
         initializeCoreMetrics(applicationContext)
 
-        // This must be set before anything that might trigger the sending of pings.
-        initialized = true
+        // This must be set before anything that might trigger the sending of pings.  We still don't
```

Comment:
Should we also guide against `initializing` already true at the top of this function?

Reply:
Good point.